src/Language/Haskell/Refact/Utils/Transform.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE CPP #-}
+module Language.Haskell.Refact.Utils.Transform
+  (
+    addSimpleImportDecl
+  , wrapInLambda
+  , wrapInPars
+  , wrapInParsWithDPs
+  , addNewLines
+  , locate
+  ) where
+
+import qualified GHC as GHC
+import qualified BasicTypes as GHC
+import qualified Data.Map as Map
+import Data.Data
+
+import Language.Haskell.GHC.ExactPrint
+import Language.Haskell.GHC.ExactPrint.Types
+
+import Language.Haskell.Refact.Utils.Monad
+import Language.Haskell.Refact.Utils.MonadFunctions
+import Language.Haskell.Refact.Utils.TypeUtils
+import Language.Haskell.Refact.Utils.Synonyms
+import Language.Haskell.Refact.Utils.ExactPrint
+
+-- The goal of this module is to provide basic transformations of the ast and
+-- annotations that are useful in multiple refactorings
+
```

Comment:
Good. The current module breakdown has been driven by the process of converting from the older HaRe to the current one.  I think this has gone far enough now that we can and should start looking at rearranging what we have, and simplifying what it does.  High level statement.